docs/test-vectors/ProductRegistrationEvidenceDocument.json

```diff
@@ -0,0 +1,162 @@
+      "model": "b835de86-27ac-4e79-ad0d-8dc1d5635fba",
+      "color": "#1a742e",
+      "material": "iste",
+      "weight": "3.166 kg",
+      "height": "3.389 m",
+      "width": "2.7 m",
+      "depth": "1.985 m",
+      "url": "https://jacey.net",
+      "isBasedOn": "https://florine.net",
+      "image": "http://placeimg.com/640/480",
+      "HTScode": "4165.98.1000"
+    }
+  ],
+  "bad": [
+    {
+      "@context": [
+        "https://w3id.org/traceability/v1"
+      ],
+      "inProductGroupWithID": "Provident dolorem quis corporis eum nesciunt autem consequatur dignissimos.",
+      "mpn": "explicabo",
+      "releaseDate": "11-2-2020",
+      "globalLocationNumber": "6918901101918",
+      "leiCode": "Apt. 730",
+      "name": "aut",
+      "description": "Id mollitia quibusdam quia. Suscipit neque voluptates placeat. Iure et quia sit iusto rerum. Aliquid ut sed velit ut voluptas sit et debitis. Eos accusantium et.",
+      "color": "#580b67",
+      "weight": "5.304 kg",
+      "height": "5.852 m",
+      "depth": "3.787 m",
+      "url": "https://margaret.org",
+      "isBasedOn": "http://elwyn.net",
+      "image": "http://placeimg.com/640/480",
+      "HTScode": "8079.89.7349",
+      "hard drive": "copy",
+      "matrix": "input",
+      "application": "index",
+      "system": "program"
+    },
+    {
+      "type": "ProductRegistrationEvidenceDocument",
+      "inProductGroupWithID": "Qui dolores illo reiciendis.",
+      "productID": "e8a749da-405f-48b9-b74c-98cac1ce114b",
+      "name": "iusto",
+      "color": "#4e215d",
+      "material": "ut",
+      "weight": "5.064 kg",
+      "width": "2.363 m",
+      "isBasedOn": "https://alberto.org",
+      "image": "http://placeimg.com/640/480",
+      "feed": "compress",
+      "alarm": "calculate",
+      "array": "override",
+      "transmitter": "copy"
+    },
+    {
+      "@context": [
+        "https://w3id.org/traceability/v1"
+      ],
+      "type": "ProductRegistrationEvidenceDocument",
+      "category": "Dolor cupiditate modi quam quidem voluptatem voluptates et.",
+      "inProductGroupWithID": "Doloremque maxime dolorem sunt est et voluptas et et.",
+      "productID": "207a22bf-7f2f-4160-ab13-efba5b8734a0",
+      "gtin": "74283373014652",
+      "releaseDate": "7-5-2020",
+      "manufacturer": "quos",
+      "globalLocationNumber": "3884833640884",
+      "name": "et",
+      "description": "Sequi soluta eum dolorum quia animi. Excepturi cum cupiditate asperiores explicabo. Laudantium sunt ut omnis iure id in laboriosam repellendus ut.",
+      "model": "b835de86-27ac-4e79-ad0d-8dc1d5635fba",
+      "color": "#1a742e",
+      "material": "iste",
+      "weight": "3.166 kg",
+      "height": "3.389 m",
+      "width": "2.7 m",
+      "depth": "1.985 m",
+      "url": "https://jacey.net",
+      "isBasedOn": "https://florine.net",
+      "image": "http://placeimg.com/640/480",
```

Comment:
```suggestion
      "url": "https://jacey.example.net",
      "isBasedOn": "https://florine.example.net",
      "image": "http://placeimg.example.com/640/480",
```

Reply:
we cannot accept changes like this on this repo, since all data is synthetically generated... you must modify the related synthetic data generator.

Reply:
@OR13 It would be helpful if direct comments like this included the handle of the person you were directing your comment to. If you're directing this to me, given the *content* of your comment, I suggest you instead direct it to the person(s) who "synthetically generated" the data or who maintain the "related synthetic data generator."

Please note that my suggested changes are absolutely appropriate, as `.*\.example\.{net|org|com|etc}` are domains reserved by IANA for such demo data, and there is no way to ensure that `{jacey|florine|placeimage}.{net|com}` will not be registered tomorrow, with DNS and files set up (accidentally or intentionally) to break all these demos and/or unable to handle the traffic unintentionally generated by these demo files.

Reply:
@TallTed agreed, @cstoecker would you consider ensuring that all domains are safe examples ? I would be open to generic solution to this that would apply to all URL / emails generated in credentials.

Reply:
since this PR is now closed, we may want to move comments to an issue.